src/utils/api.js

```diff
@@ -4,9 +4,22 @@ export const getArticles = () => {
   return axios
     .get("https://hannahtm-nc-news-backend-project.onrender.com/api/articles")
     .then((response) => {
-      return response;
+      return response.data.articles;
     })
     .catch((error) => {
       console.log(error, "ERROR");
+      return error
     });
 };
+
+export const getArticle = (article_id) => {
+  return axios
+    .get(`https://hannahtm-nc-news-backend-project.onrender.com/api/articles/${article_id}`)
```

Comment:
I think you should consider using an axios instance to create a baseURL - here are the docs on how to do that: https://axios-http.com/docs/instance